docs/Character.md

```diff
@@ -0,0 +1,14 @@
+# Character
+
+- `v::character(string $countryCode)`
+
+Validates if an input includes language specific characters.
+
+```php
+v::character('TR')->validate("Mustafa Kemal Atatürk"); //true
+v::character('TR')->validate("Ahmet"); //false
+```
```

Comment:
I will prefer some [ISO 639](http://en.wikipedia.org/wiki/ISO_639) code ([ISO 639-1](http://en.wikipedia.org/wiki/ISO_639-1) and/or [ISO 639-5](http://en.wikipedia.org/wiki/ISO_639-5)], since the relationship between language and countries is m:n.